apps/gkyl_gyrokinetic_multib_priv.h

```diff
@@ -58,6 +58,29 @@ struct gyrokinetic_multib_output_meta {
   const char *topo_file_name; // name of topology file
 };
 
+
+// field data
+struct gk_field_multib {
+  struct gkyl_gyrokinetic_multib_field info; // data for field
+  enum gkyl_gkfield_id gkfield_id; // type of field
+
+
+  struct gkyl_multib_comm_conn **mbcc_send; // comm object for allgather
+  struct gkyl_multib_comm_conn **mbcc_recv; // comm object for allgather
```

Comment:
these two have the same descriptive comment, please change.

Also, I think we should rename these with more specificity because we may need more mbcc's for other operations (e.g. perp solve), so maybe call them `mbcc_allgatherz_`